css/syntaxes.json

```diff
@@ -27,7 +27,7 @@
   "clip-source": "<url>",
   "color": "<rgb()> | <rgba()> | <hsl()> | <hsla()> | <hex-color> | <named-color> | currentcolor | <deprecated-system-color>",
   "color-stop": "<color> <length-percentage>?",
-  "color-stop-list": "<color-stop>{2,}",
+  "color-stop-list": "<color-stop>#{2,}",
```

Comment:
Since the “#” doesn’t break compatibility with actual code and examples from the -3 spec, and because it clarifies the real-world syntax, I think this is a good change.